### DISCUSSION TITLE
How beneficial is it to sandbox terminal emulators, window managers, and login managers?

### DISCUSSION THREAD

**Discussion by crocket** [netblue30/firejail#4534] [General]
I use emptty for logging into sway which is my window manager. I use terminal emulators on sway.

Is it beneficial to sandbox emptty, sway, and terminal emulators?

**Comment by rusty-snake** [comment]
You need to explain your threat model first.

General:
- Expect weird issues that a hard to debug.
- Keep in mind that sandboxes can not be nested.
